bin/cheritest/cheritest_vm.c

```diff
@@ -312,6 +313,19 @@ xfail_need_writable_tmp(const char *name __unused)
 	return (reason);
 }
 
+const char* xfail_need_writable_non_tmpfs_tmp(const char *name)
```

Comment:
style(9) calls for the returned type to be on a separate line from the function name -- i.e., you should have a carriage return before "xfail_needs_writable_non_tmpfs_tmp" here.